index.html

```diff
@@ -7,6 +7,17 @@
     <title>Portfolio Abrar Hussain</title>
     <link rel="stylesheet" href="styles.css">
     <link rel="icon" href="images/favicon.ico" type="image/x-icon">
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=GTM-WNXMSPC"></script>
+
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){window.dataLayer.push(arguments);}
+        gtag('js', new Date());
+      
+        gtag('config', 'GTM-WNXMSPC');
+      </script>
```

Comment:
- [ ] Please make sure you store data gotten from the contact form is stored in local storage as mentioned in the project requirement  🙏 .  You can learn more about how to use local storage from this [video](https://youtu.be/AUOzvFzdIk4)

Reply:
@Ugboaja-Uchechi thank you for the resource.